internal/multistorage/folder.go

```diff
@@ -137,9 +137,13 @@ func (mf Folder) GetPath() string {
 
 // GetSubFolder provides a multi-storage subfolder, which includes subfolders of all used storages.
 func (mf Folder) GetSubFolder(subFolderRelativePath string) storage.Folder {
+	newPath := path.Join(mf.path, subFolderRelativePath) + "/"
```

Comment:
We have `storage.AddDelimiterToPath / storage.JoinPath` functions, let's re-use them?

Reply:
OK, we can use `storage.AddDelimiterToPath(storage.JoinPath(mf.path, subFolderRelativePath)))` in this specific line and it will work correctly.

However, we can't drop the 3 lines below, because the functions you mentioned return "/" instead of "" in case of the root folder (other folder paths don't start with "/", though). This can be kind of unsafe to create a multistorage folder with this path: in the other comment I mentioned a problem with it in `memory.Folder` and it can be not alone. 

I was afraid to change logic of the functions you mentioned because it can potentially break something that isn't covered by tests now.

Reply:
Did you mean to replace this line only, or to drop the lines below also?

Reply:
Made it so. Please, say, if I shouldn't have done this